lib/apiservers/engine/proxy/storage_proxy.go

```diff
@@ -105,13 +105,14 @@ func NewStorageProxy(client *client.PortLayer) VicStorageProxy {
 }
 
 func (s *StorageProxy) Create(ctx context.Context, name, driverName string, volumeData, labels map[string]string) (*types.Volume, error) {
-	defer trace.End(trace.Begin(""))
+	op := trace.FromContext(ctx, "Create: %s", name)
```

Comment:
I feel this should be "VolumeCreate" to be consistent with the other volume operations, and disambiguate vs the other Creates. I don't recall if we get the stack frame when this is printed out - I'm working on the assumption not or we'd just pass "" to everything.